functions/src/config/index.js

```diff
@@ -0,0 +1,32 @@
+const dotenv = require('dotenv');
+
+process.env.NODE_ENV = process.env.NODE_ENV || 'development';
+
+const envFound = dotenv.config();
+if (!envFound) {
+	throw new Error("⚠️  Couldn't find .env file  ⚠️");
+}
+module.exports = {
+	firebaseConfig: {
+		apiKey: process.env.APIKEY,
+		authDomain: process.env.AUTHDOMAIN,
+		databaseURL: process.env.DATABASEURL,
+		projectId: process.env.PROJECT_ID,
+		storageBucket: process.env.STORAGE_BUCKET,
+		messagingSenderId: process.env.MESSAGINGSENDERID,
+		appId: process.env.APPID,
+		measurementId: process.env.MEASUREMENTID,
+	},
+	adminConfig: {
+		type: process.env.TYPE,
+		project_id: process.env.PROJECT_ID,
+		private_key_id: process.env.PRIVATE_KEY_ID,
+		private_key: process.env.PRIVATE_KEY,
+		client_email: process.env.CLIENT_EMAIL,
+		client_id: process.env.CLIENT_ID,
+		auth_uri: process.env.AUTH_URI,
+		token_uri: process.env.TOKEN_URI,
+		auth_provider_x509_cert_url: process.env.AUTH_PROVIDER_x509_CERT_URL,
+		client_x509_cert_url: process.env.CLIENT_x509_CERT_URL,
+	},
+};
```

Comment:
I think @codemutatio already created a config to handle all environment variables, so there won't be conflicts